memtier_benchmark.cpp

```diff
@@ -1094,6 +1114,10 @@ int main(int argc, char *argv[])
         fprintf(stderr, "error: data-size, data-size-list or data-size-range must be specified.\n");
         usage();
     }
+    if (cfg.taskset.is_defined() && !cfg.split) {
```

Comment:
why taskset should be used with split ? we can just do "cpuset" for all cpus in taskset ..